### PR TITLE
Installation fixes

### PR DESCRIPTION
**Pull request by axiomcura**
This PR addresses several issues related to the installation and execution of analyses from this repository on GitHub. Key updates include:

Bug Fixes: Caused the analysis to break from fresh install 
Code Cleanup: Old code has been removed 
Updated Documentation: The README file now includes an installation guide to assist users in setting up the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +45,40 @@ Below are all the notebook modules used in our study.
 | [3.jump_analysis](./notebooks/3.jump_analysis/) | Applies our model to the JUMP dataset to predict cellular injuries |
 | [4.visualizations](./notebooks/4.visualizations/) | Contains a notebook responsible for generating our figures |
 
+## Installing respoitory and dependencies
+
+This installation guide assumes that you have Conda installed. If you do not have Conda installed, please follow the documentation [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
```
Comment (jenna-tomkinson) [inline README.md]:
```suggestion
This installation guide assumes that you have Conda installed. 
If you do not have Conda installed, please follow the documentation [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
```

Comment (jenna-tomkinson) [reply README.md]: One sentence per line.

File: README.md [hunk 2]
```diff
@@ -44,6 +45,40 @@ Below are all the notebook modules used in our study.
...
+
+1. **Clone the repository**: Clone the repository into your local machine and change the directory to the repo.
+
+    ```bash
+    git clone git@github.com:WayScience/Cytotoxic-Nuisance-Metadata-Analysis.git && cd Cytotoxic-Nuisance-Metadata-Analysis
+    ```
+
+2. **Install dependencies**: There are two environment files in this repository. One is in the root project folder, containing all the necessary packages to conduct our analysis. The other is within the `./notebooks/4.visualization` directory, which includes packages for generating plots with R.
```
Comment (jenna-tomkinson) [inline README.md]: Maybe consider adding that the analysis is conducted in `Python`, but might just be something the reader should already assume.

File: README.md [hunk 3]
```diff
@@ -44,6 +45,40 @@ Below are all the notebook modules used in our study.
...
+        ```bash
+        conda env create -f ./notebooks/4.visualization/visualization_env.yaml
+        conda activate visualization-env
+        ```
+
+3. **Verify the installation**: You can check if your environment is set up correctly by listing the installed packages.
+
+    ```bash
+    conda list
+    ```
```
Comment (jenna-tomkinson) [inline README.md]: I would recommend clarifying what it means to be "set up correctly". What is the user supposed to do once they list out the packages with conda? What are they using to confirm the packages are "correct"?

File: notebooks/3.jump-analysis/3.jump_analysis.sh [hunk 4]
```diff
@@ -8,4 +8,3 @@ jupyter nbconvert --to python --output-dir=nbconverted/ *.ipynb
 
 # run the scripts
 python nbconverted/3.jump_analysis.py
-python nbconverted/3.1.overlapping_compounds.py
```
Comment (jenna-tomkinson) [inline notebooks/3.jump-analysis/3.jump_analysis.sh:11]: Is this no longer a script you need to run? Just confirming this was not included in error.

Comment (jenna-tomkinson) [reply notebooks/3.jump-analysis/3.jump_analysis.sh:11]: Is this related to removing the old code?

**Review by jenna-tomkinson — APPROVED** [review]
I am approving this PR as I don't see anything that will cause any big bugs. I added some clarifying questions and comments for you to address. 

I have one question, where in this PR do you fix a bug in the analysis from fresh install? Just confirming this in case something was left out as the only environment change is in the R environment.

File: notebooks/3.jump-analysis/3.jump_analysis.sh (hunk 4 above)
Comment (axiomcura) [reply notebooks/3.jump-analysis/3.jump_analysis.sh:11]: Correct, I removed the script but I forgot to remove the command in the .sh file

**Comment by axiomcura** [comment]
Thanks @jenna-tomkinson merging!

**Pull request merged**